.codesandbox/ci.json

```diff
@@ -25,6 +25,7 @@
     "/packages/paste-design-tokens",
     "/packages/paste-core/components/disclosure",
     "/packages/paste-core/primitives/disclosure",
+    "/packages/paste-core/components/display-pill-group",
```

Comment:
What is it that this file is helping us do?

Reply:
It's telling Codesandbox which packages to upload to it's own package manager, for our PR codesandboxes. 

Those sandboxes in each PR use the latest code on the branch, so Codesandbox uploads a temporary package so it simulates install the package from NPM. We can then use Codesandbox to very the latest packages without having to deploy them.

Each time we add a new package, we need to inform codesand of that additoin